### ISSUE TITLE
AttributeError: function 'gr_version' not found

### ISSUE THREAD

**Issue by EmDash00** [sciapp/python-gr#22]
Replaced my username with [User].

Running on Windows 10 build 18363.900

As you can see I'm using pyenv to manage my python installation. Not sure if that's causing the problem or not.

How to reproduce:

```python
import grm
```

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\[User]\.pyenv\pyenv-win\versions\3.7.5-amd64\lib\site-packages\grm\__init__.py", line 10, in <module>
    _grm = load_runtime(lib_name="libGRM")
  File "C:\Users\[User]\.pyenv\pyenv-win\versions\3.7.5-amd64\lib\site-packages\gr\runtime_helper.py", line 69, in load_runtime
    library.gr_version.argtypes = []
  File "C:\Users\[User]\.pyenv\pyenv-win\versions\3.7.5-amd64\lib\ctypes\__init__.py", line 377, in __getattr__
    func = self.__getitem__(name)
  File "C:\Users\[User]\.pyenv\pyenv-win\versions\3.7.5-amd64\lib\ctypes\__init__.py", line 382, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))
AttributeError: function 'gr_version' not found
```

**Comment by FlorianRhiem** [comment]
Hello Drason,
did you perhaps install the GR framework by itself before?
Please go to `C:\Users\[User]\.pyenv\pyenv-win\versions\3.7.5-amd64\lib\site-packages\gr\runtime_helper.py` and before line 69 (`library.gr_version.argtypes = []`) please add the following to print which library is being used:
```python
print(library_filename, library)
```

**Comment by EmDash00** [comment]
Ok, so just to note, I updated to Python 3.7.7 to see if it was just the version; however, it doesn't seem to be changing the problem.

As instructed I put

```python
print(library_filename, library)
```

on line 69. Here's the output of 

```python
import grm
```

```
C:\Users\[User]\.pyenv\pyenv-win\versions\3.7.7-amd64\lib\site-packages\gr\bin\libGR.dll <CDLL 'C:\Users\[User]\.pyenv\pyenv-win\versions\3.7.7-amd64\lib\site-packages\gr\bin\libGR.dll', handle 5f200000 at 0x2a2ab847cc8>
C:\Users\[User]\.pyenv\pyenv-win\versions\3.7.7-amd64\lib\site-packages\gr\bin\libGRM.dll <CDLL 'C:\Users\[User]\.pyenv\pyenv-win\versions\3.7.7-amd64\lib\site-packages\gr\bin\libGRM.dll', handle 5f000000 at 0x2a29b194fc8>
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\[User]\.pyenv\pyenv-win\versions\3.7.7-amd64\lib\site-packages\grm\__init__.py", line 10, in <module>
    _grm = load_runtime(lib_name="libGRM")
  File "C:\Users\[User]\.pyenv\pyenv-win\versions\3.7.7-amd64\lib\site-packages\gr\runtime_helper.py", line 70, in load_runtime
    library.gr_version.argtypes = []
  File "C:\Users\[User]\.pyenv\pyenv-win\versions\3.7.7-amd64\lib\ctypes\__init__.py", line 377, in __getattr__
    func = self.__getitem__(name)
  File "C:\Users\[User]\.pyenv\pyenv-win\versions\3.7.7-amd64\lib\ctypes\__init__.py", line 382, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))
AttributeError: function 'gr_version' not found
```

**Comment by FlorianRhiem** [comment]
We've just released version 1.15.2 which should fix this issue. We are using the same function for loading libGR and libGRM, so the libGR version check was being applied to libGRM.
